database/products.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+(function RMP() {
+  let products = { products: [] };
+
+  const getAllProducts = () => {
+    return products;
+  };
+
+  const getSpecificProduct = (req) => {
+    for (let i = 0; i <= products.products.length - 1; i++) {
+      if (parseInt(products.products[i].id) === parseInt(req)) {
+        return products.products[i];
+      }
+    }
+  };
+
+  const postProduct = (count, reqName, reqPrice, reqInventory) => {
```

Comment:
POST and PUT are route specific jargon. For databases, it's better to use variables names like create and update.